SensusService/Protocol.cs

```diff
@@ -387,6 +392,10 @@ public static void RunUnitTestingProtocol(Stream unitTestingProtocolFile)
         private bool _shareable;
         private List<PointOfInterest> _pointsOfInterest;
         private string _description;
+        private DateTime _startTime;
+        private bool _startImmediately;
+        private DateTime _endTime;
+        private bool _continueIndefinitely;
```

Comment:
If this were all behind the scenes I think so, but these are linked to On Off UI properties for user interaction. I'm not sure how to let the user choose these options otherwise, I couldn't find a way for date and time pickers to allow null values. Let me know if there's something you had in mind?